mlflow/protos/service.proto

```diff
@@ -682,9 +682,21 @@ message ListExperiments {
   // If unspecified, return only active experiments.
   optional ViewType view_type = 1;
 
+  // Maximum number of experiments desired.
+  // Servers may select a desired default value and threshold for `max_results`.
+  // Callers of this endpoint are encouraged to pass max_results explicitly and
+  // leverage page_token to iterate through experiments.
```

Comment:
@harupy @wamartin-aml Can we indicate that the max threshold is guaranteed to be at least 1000 for all servers?

Reply:
Updated the comment to indicate that!